app/views/layouts/card.html.slim

```diff
@@ -1,6 +1,6 @@
 = content_for :content do
   .container.sm-pt4
-    div class="card #{yield(:card_cls)}"
+    div class="px2 py3 sm-py5 sm-px6 mx-auto mb5 border-box card #{yield(:card_cls)}"
```

Comment:
excited to introduce you to this system! it might be the best thing invented since emojis 😉 

additional articles if you're curious -- https://blog.colepeters.com/building-and-shipping-functional-css/ https://medium.com/buzzfeed-design/how-i-learned-to-stop-worrying-and-love-the-atomic-class-98d6ccc45781#.ktwfxs78l